apps/tasks/lib/pages/settings_page.dart

```diff
@@ -0,0 +1,66 @@
+
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/constants.dart';
+
+class SettingsPage extends StatefulWidget{
+  const SettingsPage({super.key});
+
+  @override
+  State<SettingsPage> createState() => _SettingsPageState();
+}
+
+class _SettingsPageState extends State<SettingsPage> {
+  bool isDarkTheme = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(context.localization!.settings),
+      ),
+      body: ListView(
+        children: [
+          Padding(padding: const EdgeInsets.symmetric(horizontal: paddingSmall), child:
+            Column(
+              children: [
+                ListTile(
```

Comment:
Dividers are still missing, maybe `ListTile.divideTiles()` can be used

Reply:
I would like to use `ListView` here (for better optimizations).
```dart
ListView(
	children:
	ListTile.divideTiles(
    	context: context,
    	tiles: [ /* ... */ ]
  	).toList(),
),
```